app/src/main/java/com/example/pangea/AccountsActivity.kt

```diff
@@ -1,23 +1,30 @@
 package com.example.pangea
 
 import android.app.Dialog
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.util.Log
 import android.webkit.WebView
+import android.widget.Toast
+import com.facebook.FacebookSdk
 import kotlinx.android.synthetic.main.activity_accounts.*
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
-
+@Suppress("DEPRECATION")
```

Comment:
Just a question: Why do we need this? Are we using any deprecated functionality?

Reply:
yes we need that so we can use FacebookSdk.sdkInitialize()